app/domain/errors.rb

```diff
@@ -353,9 +353,11 @@ module AuthnGcp
         code: "CONJ00067E"
       )
 
-      TokenClaimNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
-        msg:  "Claim '{0-claim-name}' not found or empty in token. " \
-              "Verify that you requested the token using 'format=full'",
+      ResourceRestrictionNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
+        msg:  "Resource restriction '{0-resource-restriction-name}' does not exists " \
+            "resource in JWT token. " \
+            "Verify that you configured the host with permitted restrictions. " \
+            "In case of Compute Engine token verify that you requested the token using 'format=full' " ,
```

Comment:
msg:  "Resource restriction '{0-resource-restriction-name}' does not exist. " \
              "Make sure that you requested the token using 'format=full'",	            "resource in JWT token. " \
            "Make sure that you configured the host with permitted restrictions. " \
            "If you are working with a Compute Engine token, make sure that you requested the token using 'format=full' " ,

Reply:
@nessiLahav  - The ""resource in JWT token. " \" bit seems a bit out of place here... where does it belong? Not sure what to do with it.

Reply:
Okay, so I think I get it:
msg: "Resource restriction '{0-resource-restriction-name}' does not exist in the JWT. "
"Make sure that you requested the token using 'format=full'", and that you configured the host with permitted restrictions.  In the case of a Compute Engine token, make sure that you requested the token using 'format=full' " .

Reply:
it is not correct logically,
It should be:
"Resource restriction '{0-resource-restriction-name}' does not exist in the JWT. "
"Make sure that you configured the host with permitted restrictions. In the case of a Compute Engine token, make sure that you requested the token using 'format=full' " .

Reply:
yes that's fine